include/lcp-catlistdisplayer.php

```diff
@@ -261,7 +261,7 @@ public function get_post_title($single, $tag = null, $css_class = null,
     }
 
     // Shortcode parameters take precedence.
-    $tag = esc_attr($this->params['title_tag']) ?: esc_attr($tag);
+    $tag = tag_escape($this->params['title_tag']) ?: tag_escape($tag);
     $css_class = $this->params['title_class'] ?: $css_class;
```

Comment:
Now when I look at it again I see I forgot one thing: `$css_class` needs to be escaped here because the wrapper only escapes tags, other properties need to be passed safe. So definitely need to add `esc_attr` here.